Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonPalette.cs

```diff
@@ -321,6 +321,8 @@ public KryptonPaletteCheckButtons ButtonStyles
         #endregion
 
         #region Cue Hint
+        /*
```

Comment:
Do not do this, otherwise the CueHint text will not be stored in the designer into the relevant file

Reply:
Should I set it to `null`, as I think it's linked to #669?